shared/utils/relabel-ids.py

```diff
@@ -429,13 +433,10 @@ def verify_correct_form_of_referenced_cce_identifiers(xccdftree):
     xccdfrules = xccdftree.findall(".//{%s}Rule" % xccdf_ns)
     for rule in xccdfrules:
         identcce = _find_identcce(rule)
-        if identcce is None:
-            continue
-
-        cceid = identcce.text
-        if not ssgcommon.cce_is_valid(cceid):
-            msg = "The CCE ID {0} is not in valid form.".format(cceid)
-            raise ssgcommon.SSGError(msg)
+        if identcce is not None:
+            cceid = identcce.text
+            if not ssgcommon.cce_is_valid(cceid):
+                rule.remove(identcce)
```

Comment:
I strongly prefer when we fail build than silently remove. This PR changes behavior in a way that invalid CCEs will be ignored and only spotted by users and customers.

Reply:
Actually, it removes the idents with invalid CCEs. Failing on invalid CCEs is not a good way to go IMO as we would also need do to for all references to be consistent which means that we would be getting bogged down in build failures which we are actually getting right now with invalid CCEs. The code just never worked to show that in the first place.

Reply:
@mpreisler ^